README.md

```diff
@@ -16,165 +12,109 @@ Here you will learn how to interact with the Ubidots API using an Arduino GPRS s
 * [Ubidots Arduino GPRS library](https://github.com/ubidots/ubidots-arduino-gprs/archive/master.zip)
 * An active SIM card with a data plan 
 
+**NOTE:** This library is not compatible with the old versions. Find the last stable version (**3.0.0**) on the link below:
+
+* [Ubidots Arduino GPRS Libray - 3.0.0.](https://github.com/ubidots/ubidots-arduino-gprs/archive/3.0.0.zip)
+
 ## Setup
+1. Download the [Arduino IDE](https://www.arduino.cc/en/Main/Software).
+2. Open you IDE and click on **Sketch -> Include Library -> Manage Libraries**.
+3. In aditional look for Ubidots GPRS, and install it.
 
-1. Download the Ubidots_Arduino_GPRS.h library [here](https://github.com/ubidots/ubidots-arduino-gprs)
-2. Go to the Arduino IDE, click on **Sketch -> Include Library -> Add .ZIP Library**
-3. Select the .ZIP file of **Ubidots Arduino GPRS** and then "**Accept**" or "**Choose**"
-4. Close the Arduino IDE and open it again.
+## Alternative Setup
+1. Download the [Arduino IDE](https://www.arduino.cc/en/Main/Software).
+2. Download the **Ubidots GPRS library** [here](https://github.com/ubidots/ubidots-arduino-gprs/archive/master.zip).
+3. Now, click on **Sketch -> Include Library -> Add .ZIP Library**.
+4. Select the .ZIP file of Ubidots and then "**Accept**" or "**Choose**".
+5. Close the Arduino IDE and open it again.
+
+# Documentation
 
 Before running the examples, make sure you have an active data plan. You will also need your mobile operator’s APN settings (APN, USER, PASSWORD). You should be able to easily find these settings in Google or in your operator’s website.
```

Comment:
rephrase this

Reply:
"make sure you have an active data plan" --> make sure you a SIM Card able to connect to the cloud